Documentation/Usage.md

```diff
@@ -115,204 +115,272 @@ url.
 Example of a published package:
 https://github.com/apple/example-package-fisheryates
 
-## Require System Libraries
+## Import system libraries
 
-You can link against system libraries using the package manager. To do so, there
-needs to be a special package for each system library that contains a modulemap
-for that library. Such a wrapper package does not contain any code of its own.
+As of Swift 4.2, the package manager is capable of importing C libraries from the system without requiring a special wrapper package. We’ll import [Cairo](cairographics.org), a popular 2D vector graphics library, as an example. This guide assumes you already know how to include and link a system library in a C or C++ project.
 
-Let's see an example of using [libgit2](https://libgit2.github.com) from an
-executable.
+For our example, we’ll create a new package called `example`.
```

Comment:
Nit: please preserve standard Swift project documentation style that introduces code blocks with a colon (:) instead of period, here and below.

Reply:
i replaced the periods with colons wherever it was saying stuff like “the following:” or “this code:” the other sentences i think aren’t introducing the code blocks directly so they should have periods

Reply:
All sentences that introduce code blocks should have colons in Swift project documentation style; you may refer to [_The Swift Programming Language_](https://docs.swift.org/swift-book/LanguageGuide/TheBasics.html) for reference.

Reply:
done